ID: 34220
Title: tidb-lightning import sometimes fail: converting driver.Value type []uint8 ("9223372036854775808") to a int64: value out of range
Description:
## Bug Report

Please answer these questions before submitting your issue. Thanks!

- tiup tidb-lightning --config lightning.toml -f coms.PK* ->sucessfully

- tiup tidb-lightning --config lightning.toml -f coms.UK_*
tiup is checking updates for component tidb-lightning ...
Starting component `tidb-lightning`: /root/.tiup/components/tidb-lightning/v6.0.0/tidb-lightning /root/.tiup/components/tidb-lightning/v6.0.0/tidb-lightning --config lightning.toml -f coms.UK_*
Verbose debug logs will be written to tidb-lightning.log

Error: fetch table columns failed: sql: Scan error on column index 3, name "NEXT_GLOBAL_ROW_ID": converting driver.Value type []uint8 ("9223372036854775808") to a int64: value out of range
tidb lightning encountered error: fetch table columns failed: sql: Scan error on column index 3, name "NEXT_GLOBAL_ROW_ID": converting driver.Value type []uint8 ("9223372036854775808") to a int64: value out of range

- config:
[lightning]

level = "info"
file = "tidb-lightning.log"
#check-requirements = false

[checkpoint]
enable = true
driver = "file"

[tikv-importer]
backend = "tidb"
sorted-kv-dir = "/home/ssd/sort"

[mydumper]
data-source-dir = "s3://nfs/xxx/comp1?xxx&force-path-style=true"

[tidb]
host = "172.xxx"
port = 4000
user = "root"
password = ""
status-port = 10080
pd-addr = "172.xxx:2379"

### 1. Minimal reproduce step (Required)

<!-- a step by step guide for reproducing the bug. -->

### 2. What did you expect to see? (Required)

### 3. What did you see instead (Required)

### 4. What is your TiDB version? (Required)
6.0.0

<!-- Paste the output of SELECT tidb_version() -->

